lib/src/drawConfigContext.dart

```diff
@@ -0,0 +1,187 @@
+/// Copyright (c) 2017, the Dart Reddit API Wrapper  project authors.
```

Comment:
There's an extra space between 'Wrapper' and 'project' in some of the files (totally my fault). Mind removing it since you're here?